pkgs/development/ocaml-modules/pp/default.nix

```diff
@@ -0,0 +1,29 @@
+{ buildDunePackage
+, fetchFromGitHub
+, ppx_expect
+, lib
+}:
+
+buildDunePackage rec {
+  pname = "pp";
+  version = "1.1.2";
+
+  src = fetchFromGitHub {
```

Comment:
You can use the tarball from `https://github.com/ocaml-dune/pp/releases/download/1.1.2/pp-1.1.2.tbz` as source; it includes correct version information.

Reply:
done